deployments/liqo/crds/discovery.liqo.io_foreignclusters.yaml

```diff
@@ -68,38 +65,32 @@ spec:
                 required:
                 - clusterID
                 type: object
-              discoveryType:
-                default: Manual
-                description: How this ForeignCluster has been discovered.
-                enum:
-                - LAN
-                - WAN
-                - Manual
-                - IncomingPeering
-                type: string
-              join:
-                default: false
-                description: Enable join process to foreign cluster.
-                type: boolean
-              namespace:
-                description: Namespace where Liqo is deployed. (Deprecated)
+              foreignAuthUrl:
+                description: URL where to contact foreign Auth service.
+                pattern: https:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)
                 type: string
-              trustMode:
-                default: Unknown
-                description: Indicates if this remote cluster is trusted or not.
+              peeringEnabled:
```

Comment:
I was wondering (maybe a silly idea), would it be clearer for the user to specify the "direction" of the peering? In order to easily identify the corresponding field in the status. What do you think? @aleoli @alacuku @giorio94

Reply:
Do you mean renaming the field key or changing the values semantic? I feel the latter would be probably quite a big modification with many consequences. 

Still, I agree outgoingPeeringEnabled would be clearer, with no impact. And, as for the future, I also agree it would probably nice having the possibility to disable the incoming peering through the foreigncluster (or accept it conditionally)

Reply:
The first one, but the second looks like an interesting future improvement.

My point was to map the spec field easily with the status one, to avoid the reaction "I enabled the peering, but which field should I look to check if everything went well?".